tests/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ include_directories("${GTEST_INCLUDE_DIR}"
                     "${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}"
                     "${CMAKE_SOURCE_DIR}/include"
                     "${CMAKE_SOURCE_DIR}/src"
-                    "${CMAKE_SOURCE_DIR}/thirdparty/cnmem/include")
+                    "${CMAKE_CURRENT_SOURCE_DIR}/../include")
```

Comment:
Can we use an absolute path, here? I feel it would make the include statement more transparent.

If you need to access an include from the primary project (instead of this submodule) you should be able to use `${CMAKE_BINARY_DIR}` which would point to `/path/to/src/build` in the project which called RMM as a submodule.

Reply:
When building as the root project (not a submodule), CMAKE_BINARY_DIR will point to rmm/build, and I will need a relative (..) path to include.

When building as a submodule, CMAKE_BINARY_DIR will point to cudf/build, so to get RMM includes I would need `${CMAKE_BINARY_DIR}../thirdparty/rmm/include`, which is no less relative, and is completely wrong for the case where RMM is built as primary project.

So I don't see how I can do what you are suggesting.

The problem is that this is the tests CMakeLists.txt, which is the third level down when building RMM as a submodule of cuDF. I don't know of a way to point to the directory of the parent CMakeLists.txt with an absolute path, only to the current CMakeLists.txt and the root CMakeLists.txt. Do you?

Reply:
I see your point. I'm not sure of an easier way to do this that doesn't also break building RMM as a stand-alone library.

Reply:
Thanks, will leave it this way for now.